web/index.ts

```diff
@@ -207,14 +189,15 @@ const App = (_: any, state: AppState, setState: SetState) => {
         md = true,
         text = '**Hello** World',
         images=[imageLightOptions[0].value],
-        widths=[],
-        heights=[],
+        widths=['225'],
+        heights=['225'],
```

Comment:
I think we need to default to `auto` like we did before

Reply:
If we do so the input field will have auto too? Which might be pretty misleading? Like thinking it would allow string values to it?

Reply:
How were we handling `auto` before?

Ideally, we want the input to end up with an empty string by default so it shows the placeholder.

Right now, it isnt clear which one is width and which one is hight.

Reply:
So lets default to the empty string or rather, empty array

Reply:
Changed it to empty string. Also I see that in the current prod the heights and widths in the URL generated were sent as `undefined` when 2nd image's height or width was set (backend was adding fallback of height `225` and width `auto`). I've added default values in frontend as well (so that we don't send undefined anymore) 

When adding new image the default height and width will be set just like prod and will not be shown in input. :)